products/mozilla_org/test_mozilla_org.md

```diff
@@ -0,0 +1,67 @@
+* Locamotion: https://mozilla.locamotion.org/{locale code}/mozilla_lang/{project name}/{sub pages}
+* Web dashboard: https://l10n.mozilla-community.org/webdashboard/{local code}. Visit this page on a regular basis to check localization progress, pending work, deadline, and errors that were introduced during translation. 
+
+We highly advise you to ask other community members to conduct peer review not only on Pontoon or Locamotion, but on staging While not all the languages are required for certain projects, each community can opt in the projects at a later time.
+
+## What to test:
+
+### Pre-l10n test
+* Have your [glossary]((https://transvision.mozfr.org/) available as a reference, select mozilla.org as Repository, your language as your Target Locale.
+* Associate the page with a product or site if it is localized. (e.g.: Firefox, Test Pilot).
+* Have the matching US page up as reference, though some strings may not be identical due to A/B testing
+* Have the project you just localized available for editing (Pontoon or Locamotion)
+
+## Linguistic Testing
+* Translation quality in context
+* Grammatically correct in context
+* Check for truncation: button text remains inside the button
+* Header line break wraps at proper place
+* Text not overlapping graphic
+* Terminology is consistency with product, and between web pages
+* Brand names remain in English
+* Product names comply to Mozilla guideline and adhere to what the community has agreed to
+* No corrupted characters
+* Click on the links on the page, which should take you to the pages of the same language if they are localized?
+* Nav bar terms consistent with the page titles they are linked to 
+* Footer links don’t overlap with one another
+
+You can make linguistic changes directly in Pontoon, Locamotion or github.  
+
+## Functionality Testing
+* Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox.
+* Font support and readability
+* Footer: links point to pages of in the same language if those pages, sites or documents are localized
+* Language list: your language is listed as one of the option
+* Deliberately type a wrong broken link, verify whether 404 page is in the same language.
+* Sorting order of languages in dropdown list
+* In RTL (right to left languages), both target language and English flows in the correct directions.  The mirroring effect is correct as defined by Mozilla UX team.  Indentation and alignment are correct.
+
+
+## Testing for Fundraising Campaigns: 
+The annual fundraise campaign is localized in many languages, but not all languages. 
+* Date and time format is country specific
+* Currency availability
+* Postal/zip code in the proper format
+* Mailing address format appropriate for the country
+* Payment system available for the targeted country
+* Country list in your language and order
+
+## Compatibility testing:
```

Comment:
Is this really on localizers to test? It might be, especially for languages not using Latin characters, but it's border line.